### PR TITLE
Add victs

### PR DESCRIPTION
**Pull request by VeSeaOfficial**
Secondary commit of Victs dApp. 

Expected informative webpage: Dec. 15th

Go-Live: December 18th

### CODE REVIEW

**Comment by VeSeaOfficial** [comment]
Tagging @libotony and @XJWX89 - Submitting new PR for previously-approved branch after forced-merge. URL is now live with information.

Go-Live: Dec 18th.

**Review by XJWX89 — APPROVED** [review]

**Pull request merged**
